src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -1366,6 +1343,49 @@ bool FancyZones::ShouldProcessSnapHotkey(DWORD vkCode) noexcept
     return false;
 }
 
+void FancyZones::ApplyQuickLayout(int key) noexcept
+{
+    std::unique_lock writeLock(m_lock);
+
+    HMONITOR monitor = WorkAreaKeyFromCursor();

```

Comment:
We call 
```
HMONITOR monitor = WorkAreaKeyFromCursor();
```
only to pass `monitor` to 
```
auto workArea = m_workAreaHandler.GetWorkArea(m_currentDesktopId, monitor);
```
what about adding a method `m_workAreaHandler.GetWorkAreaFromCursor(m_currentDesktopId);`?

Reply:
I don't think we need that right now